sdk/js/src/token_bridge/__tests__/consts.ts

```diff
@@ -6,49 +6,71 @@ const ci = !!process.env.CI;
-  14, 173, 153, 4, 176, 224, 201, 111, 32, 237, 183, 185, 159, 247, 22, 161, 89,
-  84, 215, 209, 212, 137, 10, 92, 157, 49, 29, 192, 101, 164, 152, 70, 87, 65,
-  8, 174, 214, 157, 175, 126, 98, 90, 54, 24, 100, 177, 247, 77, 19, 112, 47,
-  44, 165, 109, 233, 102, 14, 86, 109, 29, 134, 145, 132, 141,
+    14, 173, 153, 4, 176, 224, 201, 111, 32, 237, 183, 185, 159, 247, 22, 161,
+    89, 84, 215, 209, 212, 137, 10, 92, 157, 49, 29, 192, 101, 164, 152, 70, 87,
+    65, 8, 174, 214, 157, 175, 126, 98, 90, 54, 24, 100, 177, 247, 77, 19, 112,
+    47, 44, 165, 109, 233, 102, 14, 86, 109, 29, 134, 145, 132, 141,
 ]);
 export const SOLANA_CORE_BRIDGE_ADDRESS =
-  "Bridge1p5gheXUvJ6jGWGeCsgPKgnE3YgdGKRVCMY9o";
+    "Bridge1p5gheXUvJ6jGWGeCsgPKgnE3YgdGKRVCMY9o";
 export const SOLANA_TOKEN_BRIDGE_ADDRESS =
-  "B6RHG3mfcckmrYN1UhmJzyS1XX3fZKbkeUcpJe9Sy3FE";
+    "B6RHG3mfcckmrYN1UhmJzyS1XX3fZKbkeUcpJe9Sy3FE";
 export const TERRA_NODE_URL = ci
-  ? "http://terra-terrad:1317"
-  : "http://localhost:1317";
+    ? "http://terra-terrad:1317"
+    : "http://localhost:1317";
 export const TERRA_CHAIN_ID = "localterra";
 export const TERRA_GAS_PRICES_URL = ci
-  ? "http://terra-fcd:3060/v1/txs/gas_prices"
-  : "http://localhost:3060/v1/txs/gas_prices";
+    ? "http://terra-fcd:3060/v1/txs/gas_prices"
+    : "http://localhost:3060/v1/txs/gas_prices";
 export const TERRA_CORE_BRIDGE_ADDRESS =
-  "terra18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5";
+    "terra18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5";
 export const TERRA_TOKEN_BRIDGE_ADDRESS =
-  "terra10pyejy66429refv3g35g2t7am0was7ya7kz2a4";
+    "terra10pyejy66429refv3g35g2t7am0was7ya7kz2a4";
 export const TERRA_PRIVATE_KEY =
-  "notice oak worry limit wrap speak medal online prefer cluster roof addict wrist behave treat actual wasp year salad speed social layer crew genius";
+    "notice oak worry limit wrap speak medal online prefer cluster roof addict wrist behave treat actual wasp year salad speed social layer crew genius";
 export const TEST_ERC20 = "0x2D8BE6BF0baA74e0A907016679CaE9190e80dD0A";
 export const TEST_SOLANA_TOKEN = "2WDq7wSs9zYrpx2kbHDA4RUTRch2CCTP6ZWaH4GNfnQQ";
 export const WORMHOLE_RPC_HOSTS = ci
-  ? ["http://guardian:7071"]
-  : ["http://localhost:7071"];
+    ? ["http://guardian:7071"]
+    : ["http://localhost:7071"];
+
+export type Environment = "devnet" | "testnet" | "mainnet";
+export const CLUSTER: Environment = "devnet" as Environment; //This is the currently selected environment.
+
+export const TERRA_HOST =
+    CLUSTER === "mainnet"
+        ? {
+              URL: "https://lcd.terra.dev",
+              chainID: "columbus-5",
+              name: "mainnet",
+          }
+        : CLUSTER === "testnet"
+        ? {
+              URL: "https://bombay-lcd.terra.dev",
+              chainID: "bombay-12",
+              name: "testnet",
+          }
+        : {
+              URL: TERRA_NODE_URL,
+              chainID: "columbus-5",
+              name: "localterra",
+          };
```

Comment:
I think what I'm seeing here is a ternary, but seems a bit overloaded.  What do you think about taking the JSON blobs out as variables so it's a bit more readable?  I'm also a n00b to ts, so tell me pound sand if this is a common pattern, but just reads a bit hard to me.

Reply:
This is a common pattern so that TERRA_HOST can be a const.  I, actually, stole this code from another location and changed one value.